evalml/pipelines/components/transformers/column_selectors.py

```diff
@@ -56,10 +59,8 @@ def fit(self, X, y=None):
         return self
 
     def transform(self, X, y=None):
-
-        if not isinstance(X, pd.DataFrame):
-            X = pd.DataFrame(X)
-
+        X = _convert_to_woodwork_structure(X)
```

Comment:
Agreed, though maybe arguably cleaner to have the woodwork conversion handle things, rather than having code check if its a woodwork or numpy structure and then calling the appropriate method to convert. 🤔